code/modules/atmospherics/gasmixtures/immutable_mixtures.dm

```diff
@@ -1,57 +1,84 @@
 //"immutable" gas mixture used for immutable calculations
 //it can be changed, but any changes will ultimately be undone before they can have any effect
 
-/datum/gas_mixture/immutable
+/datum/gas_mixture/turf/immutable
```

Comment:
Isn't repathing these going to break all other immutable gases that are dependent on these procs?

Reply:
Ah yeah I need to revert that path, maybe? I need to take a look at it.

To answer your question, the answer is no. Immutable gasses are only used for space, and I've updated that.

Reply:
For future ref, /turf/ allows heat capacity to be at vacuum levels when the turf is empty. If someone adds an empty planetary mix they should update this, otherwise we're good.